admin_guide/manage_scc.adoc

```diff
@@ -208,6 +195,77 @@ In order to preserve customized SCCs during upgrades, do not edit settings on
 the default SCCs other than priority, users, and groups.
 ====
 
+[[example-security-context-constraints]]
+=== Example Security Context Constraints Settings
+
+.Without Explicit runAsUser Setting
+----
+apiVersion: v1
+kind: Pod
+...
+fsGroup:
+  type:
+  ranges:
+  - min: 5000
+    max: 6000
+runAsUser: <1>
+  type:
+  uidRangeMin:
+  uidRangeMax:
+seLinuxContext:
+  type:
+  SELinuxOptions:
+    user: <selinux-user-name>
+    role: ...
+    type: ...
+    level: ...
+supplementalGroups:
+  type: MustRunAs
+  ranges:
+  - min: 5000
+    max: 6000
+----
+<1> When `runAsUser` is not explicitly set, one of the available SCCs (most likely
+ a restricted SCC) will process the configuration. `runAsUser` will be set to a
+ random value.  The admission controller looks for the
+ `openshift.io/sa.scc.uid-range` annotation to populate range fields. See
+ xref:../architecture/additional_concepts/authorization.adoc#understanding-pre-allocated-values-and-security-context-constraints[Understanding
+ Pre-allocated Values and Security Context Constraints] for more information.
+
+.With Explicit runAsUser Setting
+----
+apiVersion: v1
+kind: Pod
+...
+fsGroup:
+  type:
+  ranges:
+  - min: 5000
+    max: 6000
+runAsUser: <1>
+  type: RunAsAny
+  uidRangeMin: 1000100000
+  uidRangeMax: 1000100999
+seLinuxContext:
+  type:
+  SELinuxOptions:
+    user: <selinux-user-name>
+    role: ...
+    type: ...
+    level: ...
+supplementalGroups:
+  type: MustRunAs
+  ranges:
+  - min: 5000
+    max: 6000
+----
+<1> If you do not have access to a SCC that permits any UID, the `runAsUser` setting
+will be rejected by default.
```

Comment:
> If you

This could be interpreted wrong..

> to a SCC that permits any UID

and this is also vague..

> will be rejected by default

and here it's better to specify in what circumstances..

I'd suggest the following:

> A container (or a pod) that requests a specific user id, will be accepted by the OpenShift only when a service account (or a user) has been granted access to SCC that allows such user id. SCC may allow arbitrary ids, or an id that falls into a range, or even exactly this user id.

Hm, this also isn't perfect, so, please, help me improve it :)